src/middlewared/middlewared/plugins/failover.py

```diff
@@ -78,6 +78,14 @@ class Config:
         datastore_extend = 'failover.failover_extend'
         cli_namespace = 'system.failover'
 
+    ENTRY = Dict(
+        'failover_entry',
+        Int('id', required=True),
+        Bool('disabled', required=True),
+        Int('timeout', required=True),
+        Bool('master', required=True),
```

Comment:
`master` can be `null` i assume ?

Reply:
No, it always exists in the db as a boolean.

Reply:
Why do we allow it to be null on update method ?

Reply:
If it's null, then we figure out which node is to become (or stay) the master node.